docs/docs/rest-api/public/api/v2/types/deploymentEventsInfo.raml

```diff
@@ -0,0 +1,42 @@
+#%RAML 1.0 Library
+uses:
+  strings: stringTypes.raml
+  deploymentAction: deploymentAction.raml
+  deploymentPlan: deploymentPlan.raml
+
+types:
+
+  # Annotate the deployment action with more information required for the event
+  DeploymentActionInfo:
+    type: deploymentAction.DeploymentAction
+    description: A deployment action with additional status information
+    properties:
+      readinessCheckResults:
+        type: readinessCheck.TaskReadinessCheckResult[]
+        description: |
+          If there are readiness checks present for this deployment this property contains
+          their status for every task.
+
+  # Annotate the deployment plan with more information required for the event
+  DeploymentStepInfo:
```

Comment:
According to `DeploymentFormats` this should include fields `id`, `version` and `steps`.

Reply:
They are present in the base type of `deploymentPlan.DeploymentPlan`

Reply:
As I've mentioned above I would not express such a relation in the RAML.